backend/Watchdog.NetCore.Common/WatchdogSettingsBase.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Watchdog.NetCore.Common
+{
+    public abstract class WatchdogSettingsBase
+    {
+        internal const string _defaultApiEndPoint = "https://bsa-watchdog.westeurope.cloudapp.azure.com/collector/issues/client";
+        
```

Comment:
Not sure if it's a good idea to store this here.

Reply:
This class is put into a library that will be used as a NuGet package in the future and we, as developers, will ensure that this link is up-to-date (besides, I don't think there will be any router changes in production).

Reply:
Ok, but why that const have internal modifier?

Reply:
To be able to interact with this variable in the assembly, but not outside of it.

Reply:
Then I think it's better to do it with a internal property.

Reply:
This field will not be used in any other assemblies because it has an internal access modifier. This field cannot be modified because it is a constant. I see no reason to replace it with a property.